docker-stack-prod.yml

```diff
@@ -45,7 +45,7 @@ services:
             - back_network
         volumes:
             - /var/run/docker.sock:/var/run/docker.sock
-            - /volumes/whitebox_program_uploads:/uploads:ro
```

Comment:
The 'compilations' folder does not need to be created by the docker at execution time. It is created at build time by executing the the following line of the Makefile : 

https://github.com/CryptoExperts/wb_contest_submission_server/blob/0f5623aba84f271fa449eac95d4d7c131c12602c/Makefile#L41

 So we do want to keep the read-only access for the launcher docker.